common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -67,6 +68,18 @@ def _goto_problem_page(self):
         self.problem_page = ProblemPage(self.browser)
         self.assertEqual(self.problem_page.problem_name, 'TEST PROBLEM 1')
 
+    def _change_problem_release_date_in_studio(self):
+        """
+        Change problem release date.
+        """
+        self.course_outline.q(css=".subsection-header-actions .configure-button").first.click()
+        self.course_outline.q(css="#start_date").fill("01/01/2030")
+        self.course_outline.q(css=".action-save").first.click()
```

Comment:
How we ended up with this method ? I don't remember this method as part of bookmarks epic.